lib/figma.js

```diff
@@ -0,0 +1,65 @@
+const Figma = require('figma-js');
+
+module.exports =  {
+    /**
+     * Returns a client to use the Figma API
+     * @param {string} token user token 
+     * @return {Object} 
+     */
+    getClient: (token) => Figma.Client({personalAccessToken: token}),
```

Comment:
Just name your arg `personalAccessToken` here and do `Client({ personalAccessToken })`